### DISCUSSION TITLE
Schrodinger's smoke 2D implementation

### DISCUSSION THREAD

**Discussion by xingjianhou** [Part-II-Computational-Physics/.github#97] [General]
For Schrodinger's smoke project, the prompt suggested we try 2D implementations first as proof of principle. However, I started with 3D and somehow managed to do it. Is the 2D case a mandatory/important part of the project, or is it just meant to help make the code work? Would I be marked against for not mentioning 2D implementations in the report?

**Comment by AustenLamacraft** [comment]
Certainly not!
